reports-backend/src/main/java/org/jboss/da/listings/api/model/ProductArtifactRelationship.java

```diff
@@ -0,0 +1,26 @@
+package org.jboss.da.listings.api.model;
+
+import lombok.AccessLevel;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import lombok.ToString;
+
+@NoArgsConstructor(access = AccessLevel.PUBLIC)
+@ToString
+public class ProductArtifactRelationship {
```

Comment:
You are correct. We need this entity. But it should be @Entity and have an id column. Or do I miss anything?